InterviewQuestions/Chapters/ArraysStrings/ArraysAndStringSolutions.cs

```diff
@@ -65,5 +67,26 @@ public string stringCompressionAlternate(string input)
             }
             return (input.Length<compressedString.Length) ? input : compressedString; 
         }
+        public bool isStringPermutation(string input,string testInput)

```

Comment:
maybe you could name arguments as "firstString ,secondString" to make it more readeble.

Reply:
all done. Also you can see here e7a67959570ed2c6ca91173c43c3510bd4bcc39c